solutions/landing-zone/README.md

```diff
@@ -172,9 +162,22 @@ To deploy this Landing Zone you will first need to create a Bootstrap project wi
 
     **Note on Project IDs**: All IDs should be universally unique, Must be 6 to 30 characters in length, can only contain lowercase letters, numbers, and hyphens. Must start with a letter. Cannot end with a hyphen. Cannot be in use or previously used; this includes deleted projects. Cannot contain restricted strings, such as google and ssl.
 
-5. Deploy
+   ## Deployment
+
+    This solution can be deployed in a few different ways depending on the behavior you want. 
+    - kpt
+    - GitOps
+      - [Git](#git)
+      - [OCI](#oci)
 
-    a. kpt
+
+    Deploying through `kpt` is a more traditial push approach to deploying the infrastructure. This will deploy the infrastructure resources into the target cluster similar to how `kubectl` operates. Some of the deployment advantages `kpt` offers is the ability to use functions live `render` to populate variables in the configuration and `gatekeeper` which allows us to validate policy before deployment. 
+
+    When deploying resources `kpt` will wait for all resources to deploy before finishing, `kpt` also provides annotations such as [depends-on](https://kpt.dev/reference/annotations/depends-on/) that allow us to order the deployment of resources. 
+
+    GitOps allows us to connect our cluster to a "source of truth" and deploy from that. To do this we use [Anthos Config Management](https://cloud.google.com/anthos/config-management), which is pre-installed in Config Controller, and this allows us to target either a [Git Repository](https://cloud.google.com/anthos-config-management/docs/concepts/configs) or [OCI Artifact](https://cloud.google.com/anthos-config-management/docs/how-to/publish-config-registry) (Container). The benefit to this method is we now have an additional reconciliation process to ensure the deployed state matches our desired state or "source of truth". This also allows for improved automation and removes the need to run `kpt live apply` to deploy resources.
+
+    ### kpt
     
     ```
     kpt fn render
```

Comment:
may link to one of the solutions - the lz for the full 3 command cycle to fill in the template vars

Reply:
never mind - it looks like the following will come in once this branch is synced with main

```
kpt fn render landing-zone
```

Reply:
updated to be `kpt fn render landing-zone`